lib/ClangImporter/ClangAdapter.h

```diff
@@ -60,6 +60,16 @@ struct PlatformAvailability;
 Optional<const clang::Decl *>
 getDefinitionForClangTypeDecl(const clang::Decl *D);
 
+/// Returns the first redeclaration of \p D outside of a function.
+///
+/// If there are no such redeclarations, returns null.
+/// If \p D is not redeclarable, just returns \p D itself.
```

Comment:
What does "is not redeclarable" mean -- which declarations are redeclarable and which ones are not? Is this a well-known thing? :confused:

Reply:
It's Clang terminology, but clearly not clear enough.